Modules/_ctypes/callproc.c

```diff
@@ -498,9 +498,17 @@ PyCArg_repr(PyCArgObject *self)
         break;
 
     case 'c':
-        sprintf(buffer, "<cparam '%c' (%c)>",
-            self->tag, self->value.c);
-        break;
+        if ((unsigned char)self->value.c < 128
+            && _PyUnicode_IsPrintable(self->value.c)
+            && self->value.c != '\'')
```

Comment:
Is the goal here to make the output look like a syntactically valid Python string? If so, you should also exclude backslash.

Reply:
Good catch. Yes, the goal here to make the output look like a syntactically valid Python or C.